### DISCUSSION TITLE
CHIA Farmer destroyed one of my Plotter Drives today - "chia plots check" kills drive 6TB 51 plots

### DISCUSSION THREAD

**Discussion by room101-dev** [Chia-Network/chia-blockchain#6051] [General]
I have lost two drives in the past month, the first was an NVME burned up from plotting, then I tried to plot on a WD HDD blue 1TB, and it burned up that drive.

But today was really weird I was running the farmer "chia start farmer -r" and ti was always 'chia farm summary' return zero data, and all was synced, I left it last night, this am the drive was gone ( plot drive ) containing 51 plots 6TB purple WD.

So now I know the 'farmer' can destroy HHD drives, I thought it was just plotting that is abusive, turns out that the farmer runs 'chia plots check' a program that validates plots, I mean this is bullshit, we just made the plots with 'chia plots create', why in the hell would a plot need to be validated, well what the validator does is read the plot 30x times an does computation, essentially just like plotting it killed the drive. It blew out the home sector, so the drive was not recognized by the OS, I was able to get back the 'ext' format using foremost ( a tool to recover dead hdd's ), but the drive is now damaged.

So now we know the 'farmer' is just as abusive as the 'plotter' the entire chia system is just designed to kill disk drives.

I don't really care,  feel bad to lose 51 plots, but the drive is a cost of biz, I was already just doing hpool on my plots and earning xch all along, but I wanted to have just one system to test, and to get chia-net working as stand-alone, months ago it worked, the version I was using today was 1.16dev0, but now 1.17

Another new problem is that chia now requires the plot-check to be under 5 seconds, which means super fast drive, and no network drives, I already knew this from plotting. That I can't have the plot creation deliver the final to samba, I must manually pool drives and take them to the hpool miners.

**Comment by grasuu** [comment]
Honestly -- you can't blame chia here. What you experienced is called an "infant mortality" failure, you can find a good explanation here:
https://en.wikipedia.org/wiki/Bathtub_curve

It is best practice (for system admins) to "burn in" your disks -- upfront using a disk I do a 24h to 48h heavy random access script, a full badblocks run with verify and finally long and short SMART tests. And other people would call that hasty :)

**Reply by happycouak** [reply]
You dont need any "super fast drive": non-ssd drive has ~20ms worst case disk access latency, added the ~1ms of the local network I clearly don't see any issue with the 5sec limitation.

FYI all my plots are hosted on NFS servers on a 1g consumers switches, responses times are mostly under 0,2 seconds.

Last but not the least, blaming chia for killing your drive is like blaming bitcoins to consume power: that is a non sens, please be honest with yourself.
